qucs/qucs/wirelabel.cpp

```diff
@@ -106,22 +106,25 @@ bool WireLabel::getSelected(int x, int y)
 // ----------------------------------------------------------------
 void WireLabel::paint(ViewPainter *p)
 {
-  QFont font = p->Painter->font();
+  QFont f = p->Painter->font(); // save current font
+  QFont newFont = f;
+
   if (isHighlighted)
   {
 //    QColor highlightfill (Qt::blue);
 //    highlightfill.setAlpha(50);
 //    p->fillRect(x1-1, y1-1, x2, y2, highlightfill);
     p->Painter->setPen(QPen(Qt::darkBlue,3));
-    font.setWeight (QFont::DemiBold);
+    newFont.setWeight (QFont::DemiBold);
```

Comment:
Can we change for QFont::Bold?
On my Mac the font change is not visible. Probably related to http://stackoverflow.com/questions/14901933/is-it-possible-to-specify-more-than-just-regular-and-bold-versions-of-a-font